modules/installation-approve-csrs.adoc

```diff
@@ -129,12 +129,35 @@ $ oc adm certificate approve <csr_name> <1>
 ----
 $ oc get csr -o go-template='{{range .items}}{{if not .status}}{{.metadata.name}}{{"\n"}}{{end}}{{end}}' | xargs oc adm certificate approve
 ----
-
++
 [NOTE]
 ====
 Some Operators might not become available until some CSRs are approved.
 ====
 
+. When all client and server CSRs have been approved, the machines should have the `Ready` status. Verify this by running the following command:
```

Comment:
A couple of small wording suggestions for tense:

After all client and server CSRs have been approved, the machines have the `Ready` status.